### PR TITLE
replace references to collection with catalog

### LINKED ISSUE #456: Update Documentation to use new tutorial.py module for grabbing catalog URLs
We now have curated small catalogs in the `tutorial-catalogs` directory from #454 and #455 that can be called via `intake_esm.tutorial.get_url()`. The documentation should be updated to use this method for some demonstrations.

In the same effort, the documentation can be updated to use "catalog" consistently (without misnamed references to "collections").

### PR DESCRIPTION
**Pull request by jukent**
## Change Summary

Updating documentation to be more consistent in references to catalog vs collection.

## Related issue number

- Closes #456 

## Checklist

- [ ] Unit tests for the changes exist
- [ ] Tests pass on CI
- [ ] Documentation reflects the changes where applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,35 +33,35 @@ providing necessary functionality for searching, discovering, data access/loadin
...
-  (i.e., which experiment, model, the come from). The collection JSON file can be stored on a local filesystem
+  (i.e., which experiment, model, the come from). The catalog JSON file can be stored on a local filesystem
   or can be hosted on a remote server.
 
   ```python
 
   In [1]: import intake
 
-  In [2]: col_url = "https://gist.githubusercontent.com/andersy005/7f416e57acd8319b20fc2b88d129d2b8/raw/987b4b336d1a8a4f9abec95c23eed3bd7c63c80e/pangeo-gcp-subset.json"
+  In [2]: cat_url = "https://gist.githubusercontent.com/andersy005/7f416e57acd8319b20fc2b88d129d2b8/raw/987b4b336d1a8a4f9abec95c23eed3bd7c63c80e/pangeo-gcp-subset.json"
```
Comment (andersy005) [inline README.md:47]: Let's change this to use the new sample/tutorial catalogs

Comment (jukent) [reply README.md]: Done

**Review by andersy005 — APPROVED** [review]
Thank you, @jukent 🚀

**Comment by jukent** [comment]
@andersy005 Should I do a similar effort to be very intentional about the term "collection" in the `intake` repository?

**Comment by andersy005** [comment]
Can you point me to these "collections" references in the `intake` repository?

**Comment by andersy005** [comment]
good catch! that page is definitely out-of-date and an update would be great.

**Pull request merged**